psi4/src/psi4/libmints/oeprop.cc

```diff
@@ -931,45 +931,65 @@ MultipolePropCalc::MultipoleOutputType MultipolePropCalc::compute_multipoles(int
         for (iter = mp_ints.begin(); iter != mp_ints.end(); ++iter) iter->get()->print();
     }
 
-    SharedVector nuclear_contributions = MultipoleInt::nuclear_contribution(mol, order, origin_);
+    auto nuclear_contributions = MultipoleInt::nuclear_contribution(mol, order, origin_);
 
+    // => Print and populate the output tuple <=
     if (print_output) {
         outfile->Printf("\n%s Multipole Moments:\n", transition ? "Transition" : "");
         outfile->Printf("\n ------------------------------------------------------------------------------------\n");
         outfile->Printf("     Multipole             Electric (a.u.)       Nuclear  (a.u.)        Total (a.u.)\n");
```

Comment:
Electronic sound better to my ear than electric, but not a big deal either way
```suggestion
        outfile->Printf("     Multipole            Electronic (a.u.)      Nuclear  (a.u.)        Total (a.u.)\n");
```

Reply:
Last time I checked, m-poles are inherently electric but not necessarily electronic. Good catch.